pyirf/io/gadf.py

```diff
@@ -325,3 +309,177 @@ def create_rad_max_hdu(
     _add_header_cards(header, **header_cards)
 
     return BinTableHDU(rad_max_table, header=header, name=extname)
+
+
+def compare_irf_cuts(files, extname):
+    """
+    Reads in a list of IRF files and checks if the same cuts have been applied in all of them
+
+    Parameters
+    ----------
+    files: list of strings
+        files to be read
+    extname: string
+        name of the extension with cut values to read the data from in fits file
+
+    Returns
+    -------
+    match: Boolean
+        if the cuts are the same in all the files
+    """
+    with fits.open(files[0]) as hdul0:
+        data0 = hdul0['THETA_CUTS'].data
+
+    for file_name in files[1:]:
+        with fits.open(file_name) as hdul:
+            data = hdul['THETA_CUTS'].data
+            if (data != data0).any():
+                print("difference between file: " + files[0] + " and " + file_name + " in cut values: " + extname)
+                return False
+
+    return True
+
+
+def read_fits_bins_lo_hi(file_name, extname, tag):
```

Comment:
the function is needed for reading in IRFs 
this function has been planned as an IO function and as a part of reading the data in it checks that the extensions that are read from all the first are the same. 
Each extension is read only once from the FITS file. I wanted to make it simple yet general and the problem is that for different IRFs you need different extension. Since those IO operations will be very rare the overhead of read operation would be negligible. Actually I think read is not even doing the actual read, but just linking the table. 
Still, if you think that this can be a problem I can pass a list of extensions to this function.

Reply:
Please see my comment about only reading the table once. It is not necessary to read the whole table for just extracting the bin operation.

Reply:
I modified the function to read all the bins in one go